descriptions.md

```diff
@@ -160,7 +177,16 @@ Creates Ably [`TokenRequest`]{@link TokenRequest} objects and obtains Ably Token
 || `AuthOptions` ||| An [`AuthOptions`]{@link AuthOptions} object. |
 ||| `TokenDetails` || A [`TokenDetails`]{@link TokenDetails} object. |
 |`TokenDetails`: TokenDetails?||| RSA16 | A [`TokenDetails`]{@link TokenDetails} object. |
-
+| revokeTokens(TokenRevocationTargetSpecifier, issuedBefore number?, allowReauthMargin boolean?) => io `BatchResult<TokenRevocationSuccessResult \| TokenRevocationFailureResult>`  ||| RSA17 | Revokes the tokens specified by the provided [`TokenRevocationTargetSpecifier`]{@link TokenRevocationTargetSpecifier}. Only tokens issued by an API key that had revocable tokens enabled before the token was issued can be revoked. See the [token revocation docs](https://ably.com/docs/core-features/authentication#token-revocation) for more information. |
+|| `TokenRevocationTargetSpecifier` ||| A [`TokenRevocationTargetSpecifier`]{@link TokenRevocationTargetSpecifier} object. |
+|| `issuedBefore` ||| A Unix timestamp in milliseconds where only tokens issued before this time are revoked. The default is the current time. Requests with an `issuedBefore` in the future, or more than an hour in the past, will be rejected. |
+|| `allowReauthMargin` ||| If true, permits a token renewal cycle to take place without needing established connections to be dropped, by postponing enforcement to 30 seconds in the future, and sending any existing connections a hint to obtain (and upgrade the connection to use) a new token. The default is `false`, meaning that the effect is near-immediate. |
+||| `BatchResult<TokenRevocationSuccessResult \| TokenRevocationFailureResult>` || A [`BatchResult`]{@link BatchResult} containing information about the result of the token revocation request.
```

Comment:
It's not clear to me how to interpret this result — we pass a single specifier but receive a `BatchResult` with a `results` array. What do the contents of this array represent? We probably need to document that.

Reply:
This single-specifier variant has since been removed from the spec.